packages/amazonq/src/extensionShared.ts

```diff
@@ -106,7 +106,7 @@ export async function activateShared(context: vscode.ExtensionContext, isWeb: bo
     registerGenericCommands(context, contextPrefix)
 
     // Amazon Q specific commands
-    registerCommands(context)
+    // registerCommands(context)
```

Comment:
Should it remain commented? is so we can remove it i believe since we're using a version control to keep track of these :)